Dockerfile

```diff
@@ -0,0 +1,48 @@
+# RethinkRobotics/baxter-simulator:kinetic
```

Comment:
Do you have a link or tutorial that helped you set this up? I'd like to do it for the whole SDK...